docs/common-tasks/stopping-containers-and-cleanup.md

```diff
@@ -22,4 +22,9 @@ If you only want to clean up project specific stopped containers, you can run: `
 
 When you are finished with a project, if you used any persistent data storage you'll want to run a command to clean it 
 up. The exact directory to request removal from will depend on your project (see suggested directory naming guidelines in 
-[Key Concepts](../project-setup/key-concepts.md#persistent-data-volume)): `docker ssh dev sudo rm -rf /data/[project]/`
+[Key Concepts](../project-setup/key-concepts.md#persistent-data-volume)): 
+
+```
+docker-machine ssh dev
+sudo rm -rf /data/[project]/
```

Comment:
Would this be cleaner as a single command?

`docker-machine ssh dev sudo rm -Rf /data/[projectname]/`